python/langsmith/client.py

```diff
@@ -701,6 +704,19 @@ def info(self) -> ls_schemas.LangSmithInfo:
                 self._info = ls_schemas.LangSmithInfo()
         return self._info
 
+    def _get_settings(self) -> dict:
+        """Get the settings for the current tenant.
+
+        Returns:
+            dict: The settings for the current tenant.
+        """
+        if self._settings is None:
+            response = self.request_with_retries("GET", "/settings")
+            settings: dict = response.json()
```

Comment:
would be kinda nice to have a better error with `ls_utils.raise_for_status_with_text(response)`